src/cgame/rocket/rocketConditionalElement.h

```diff
@@ -90,7 +90,26 @@ class RocketConditionalElement : public Rml::Element
 
 	virtual void OnUpdate()
 	{
-		if ( dirty_value || ( !cvar.empty() && cvar_value != Cvar::GetValue( cvar.c_str() ).c_str() ) )
+		Rml::String value;
+		bool modified = false;
+
+		if ( dirty_value )
+		{
+			value = Cvar::GetValue( cvar ).c_str();
```

Comment:
c_str() unnecessary now. Is a remnant of libRocket

Reply:
Removed.